src/_includes/graphql/downloadable-items-links.md

```diff
@@ -0,0 +1,9 @@
+#### DownloadableItemsLinks attributes {#DownloadableItemsLinks}
+
+The `DownloadableProductLinks` object defines characteristics of a downloadable product.
```

Comment:
I think this should be `DownloadableItemsLinks`. @keharper you can double check.